backend/src/main/java/com/capstone/moneytree/controller/AlpacaController.java

```diff
@@ -89,6 +89,17 @@ public AlpacaController(MarketInteractionsFacade marketInteractionsFacade, SimpM
       return ResponseEntity.ok(portfolioHistory);
    }
 
+   /**
+    *  Gets the market status (open/closed).
+    *
+    * @return market status.
+    */
+   @GetMapping("/market-status/{userId}")
+   public ResponseEntity<Clock> getMarketClock(@PathVariable(name = "userId") @Valid @NotBlank String userId) {
+      Clock marketClock = marketInteractionsFacade.getMarketClock(userId);
```

Comment:
Before implementation of the websocket, we did not have the userId as a parameter for getClock method, but now I see why Alessandro added it, he needed to create the alpaca session using that Id. The market-status is not dependent on a user, so I think we can keep the getclock method with its parameter, but maybe we can remove the {userId} from the path as it can be confusing. Let me know about your decision and I can reflect the changes in the frontend if you like.

Reply:
I don't know much about that endpoint. I had in mind that AlpacaAPI would complain if no oAuthToken was attached to it. I'd have to try without the oAuthToken and see if it still does the job. All the other endpoint requires a user ID though.

Reply:
I created that endpoint, and it was working with only the Alpaca API key and the Alpaca Secret Key (tried it both on postman and in our app). More information is available here:
https://alpaca.markets/docs/api-documentation/api-v2/clock/

Reply:
Sorry David, you were right, I see that it is not working anymore without the Alpaca session, so please keep what you have, and mark this as resolved, thanks.

Reply:
I will. No worries!